<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MitgliedDialog</class>
 <widget class="QWidget" name="MitgliedDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="firstnameLabel">
       <property name="text">
        <string>Firstname</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="firstnameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lastnameLabel">
       <property name="text">
        <string>Lastname</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lastnameLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="sexLabel">
       <property name="text">
        <string>Sex</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="sexComboBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="typComboBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="typLabel">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="nationalityComboBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="nationalityLabel">
       <property name="text">
        <string>Nationality</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDateEdit" name="birthdayDateEdit"/>
     </item>
     <item row="8" column="1">
      <widget class="QDateEdit" name="membersinceDateEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="birthdayLabel">
       <property name="text">
        <string>Birthday</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="membersiceLabel">
       <property name="text">
        <string>Member since</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Speichern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delButton">
       <property name="text">
        <string>LÃ¶schen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
